app/src/views/private/components/render-template/render-template.vue

```diff
@@ -2,7 +2,7 @@
 	<div ref="templateEl" class="render-template">
 		<span class="vertical-aligner" />
 		<template v-for="(part, index) in parts" :key="index">
-			<value-null v-if="part === null || part.value === null" />
+			<value-null v-if="part === null || (typeof part === 'object' && part.value === null)" />
```

Comment:
```suggestion
			<value-null v-if="part === null || part?.value === null" />
```

Reply:
At this point in the template, we are not sure `part` is an object. It seems safer and explicit to keep typeof checking here. It's done on line 8 too.

Reply:
Good point. 
This makes me thinking if we should use more restrictive type checking. As for now, let's keep it that way.

Reply:
[vue-tsc](https://www.npmjs.com/package/vue-tsc) can run tsc typecheck in Vite/Vue3 projects.